### ISSUE TITLE
[Suggestion] - Security code - player variable to detect if admin has enable godmode

### ISSUE THREAD

**Issue by ScaRR-ZA** [EpochModTeam/Epoch#525]
I have implemented my own stats for player kills which is logged via the EntityKilled event handler. 
This is used to show how many players have killed sappers, AI etc and creates a type of leaderboard. One issue I have is that I don't want to register kills for admin players that have godmode enabled and thought it will be useful to know, and a possible way would be to add a player variable that indicates this. 
Thanks

**Comment by vbawol** [comment]
this commit adds a variable that can be used to determine if a player has god mode enabled, and should be available in next push 0.3.9 b557
https://github.com/EpochModTeam/Epoch/commit/e9958bc5d8ad43db9202d9b309d944d3147d833

Usage:

```
_gm_enabled = player getVariable ['EPOCH_gm_enabled',false];
```

**Comment by StianKruger** [comment]
Thank you very much :+1:
